src/options.json

```diff
@@ -642,6 +642,20 @@
           "namespace": "js"
         }
       },
+      "bracket_spacing": {
+        "type": "boolean",
+        "default": false,
+        "description": "Insert spaces between brackets in object literals (Supported by Coffee Formatter)",
+        "title": "Bracket spacing",
+        "beautifiers": [
+          "Coffee Formatter"
```

Comment:
This is strange. Should be Prettier.

Reply:
That's under the CoffeeScript language, which falls back to JavaScript.